Traceback (most recent call last):
  File "D:\users\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 64, in single_nb_execution
    **kwargs,
  File "D:\users\anaconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "D:\users\anaconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "D:\users\anaconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "D:\users\anaconda3\lib\asyncio\base_events.py", line 579, in run_until_complete
    return future.result()
  File "D:\users\anaconda3\lib\site-packages\nbclient\client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "D:\users\anaconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "D:\users\anaconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# This is a guess the number game.
import random
myName = input('Hello! What is your name? ')
number = random.randint(1,20)
print ( f'Well, {myName} I am thinking of a number between 1 and 20.')

guess = int(input('Take a guess.'))

if guess> number:
    print('Your guess is too high')
elif guess < number:
    print('Your guess is too  low')
else:
    print ( f'Good job, {myName}! You guessed my number!')
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
[1;32m<ipython-input-4-274b55522267>[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [1;31m# This is a guess the number game.[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[0;32m      2[0m [1;32mimport[0m [0mrandom[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 3[1;33m [0mmyName[0m [1;33m=[0m [0minput[0m[1;33m([0m[1;34m'Hello! What is your name? '[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      4[0m [0mnumber[0m [1;33m=[0m [0mrandom[0m[1;33m.[0m[0mrandint[0m[1;33m([0m[1;36m1[0m[1;33m,[0m[1;36m20[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      5[0m [0mprint[0m [1;33m([0m [1;34mf'Well, {myName} I am thinking of a number between 1 and 20.'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32mD:\users\anaconda3\lib\site-packages\ipykernel\kernelbase.py[0m in [0;36mraw_input[1;34m(self, prompt)[0m
[0;32m    853[0m         [1;32mif[0m [1;32mnot[0m [0mself[0m[1;33m.[0m[0m_allow_stdin[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m    854[0m             raise StdinNotImplementedError(
[1;32m--> 855[1;33m                 [1;34m"raw_input was called, but this frontend does not support input requests."[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    856[0m             )
[0;32m    857[0m         return self._input_request(str(prompt),

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

